bindings_ffi/src/mls.rs

```diff
@@ -2989,49 +2995,80 @@ mod tests {
         [2u8; 32]
     }
 
-    async fn register_client(inbox_owner: &LocalWalletInboxOwner, client: &FfiXmtpClient) {
-        register_client_no_panic(inbox_owner, client).await.unwrap()
+    async fn register_client_with_wallet(wallet: &FfiWalletInboxOwner, client: &FfiXmtpClient) {
+        register_client_with_wallet_no_panic(wallet, client)
+            .await
+            .unwrap()
     }
 
-    async fn register_client_no_panic(
-        inbox_owner: &LocalWalletInboxOwner,
+    async fn register_client_with_wallet_no_panic(
+        wallet: &FfiWalletInboxOwner,
         client: &FfiXmtpClient,
     ) -> Result<(), GenericError> {
         let signature_request = client.signature_request().unwrap();
+
         signature_request
             .add_ecdsa_signature(
-                inbox_owner
+                wallet
                     .sign(signature_request.signature_text().await.unwrap())
                     .unwrap(),
             )
             .await?;
+
+        client.register_identity(signature_request).await?;
+
+        Ok(())
+    }
+
+    async fn register_client_with_passkey(passkey: &PasskeyUser, client: &FfiXmtpClient) {
```

Comment:
what's the advantage of the `no_panic` version?

Also the convention for this is to name the function `register_client_with_passkey_unchecked`

Reply:
This is in `mod tests`